libsolidity/ast/Types.cpp

```diff
@@ -3811,9 +3812,22 @@ MemberList::MemberMap TypeType::nativeMembers(ASTNode const* _currentScope) cons
 			)
 		);
 	}
+	else if (auto const* arrayType = dynamic_cast<ArrayType const*>(m_actualType);
```

Comment:
```suggestion
	else if (
		auto const* arrayType = dynamic_cast<ArrayType const*>(m_actualType);
```
(If you have a line break inside a pair of parentheses, there also needs to be a line break right after the opening parenthesis and right before the closing one)